thirdeye/thirdeye-frontend/app/pods/rootcause-aggregates-cache/service.js

```diff
@@ -76,6 +76,8 @@ export default Ember.Service.extend({
   _extractAggregates(incoming, urn) {
     // NOTE: only supports single time range
     const aggregates = {};
+    aggregates[urn] = parseFloat('NaN'); // default
```

Comment:
can't we just set it to NaN ? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN

Reply:
actually Number.NaN seems more correct https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/NaN

Reply:
fixed